config/prod/bwhite-iawg.yaml

```diff
@@ -1,6 +1,10 @@
 # Provision an EC2 instance
 template_path: remote/managed-ec2-linux-v1.j2
 stack_name: bwhite-iawg
+sceptre_user_data:
+  Distro: "aws"     # (valid values: aws, ubuntu) AMIId must match distro
```

Comment:
Did you want to use amazon linux 2 or ubuntu?

Reply:
Thanks--this is for a different project. But, yes, let me switch to ubuntu since I'm more familiar with that.

Reply:
Ya just do `Distro: "ubuntu"` And make sure you update the AMIId below